nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -494,6 +494,46 @@ The ShellUserGroupProvider has the following properties:
 
 Like LdapUserGroupProvider, the ShellUserGroupProvider is commented out in the _authorizers.xml_ file.  Refer to that comment for usage examples.
 
+==== AzureGraphUserGroupProvider
+
+The AzureGraphUserGroupProvider fetches users and group details from Azure Activy Driectory (AAD) using Azure graph rest-api & SDK.
+
+This provider first fetches the list of the groups based on group filter specification (e.g. prefix, suffix, substring of group names, or list of group names).
+The group filter conditions are evaluated against _displayName_ property of Group directory object. From the list of groups, this provider loads member users of these groups.
```

Comment:
Can we make this reference the specific configuration names once we settle on them? For example:
```suggestion
This provider fetches a subset of users and groups based on the filters configured in _authorizers.xml_.
It first fetches a list of groups based on the configured group filters (`Group Filter Prefix`, `Group Filter Suffix`, `Group Filter Substring`, and `Group Filter List`).
These filter conditions are evaluated against the _displayName_ property of Azure AD Group directory object. From the list of groups, it then loads member users of these groups.
```